plenum/bls/bls_bft_replica_plenum.py

```diff
@@ -115,6 +153,28 @@ def process_commit(self, commit: Commit, sender):
             self._signatures[key_3PC] = {}
         self._signatures[key_3PC][self.get_node_name(sender)] = commit.blsSig
 
+        if f.BLS_SIGS.nm not in commit:
```

Comment:
Can it be that `commit.blsSig is None` (so we called `return`), but `commit.blsSigs is not None` ?